activetable/static/js/src/activetable.js

```diff
@@ -138,16 +138,22 @@ function ActiveTableXBlock(runtime, element, init_args) {
     }
 
     function createCell(oldCell, index){
-        var cellId = oldCell.id
-        var type = $(oldCell).find("input").attr("type");
-        var height = $(oldCell).find("input").outerHeight();
+        var cellId = oldCell.id;
+        var kind = "textarea";
```

Comment:
kind? what does it mean? @andrey-canon please, be more specific with variable names.